README.md

```diff
@@ -1,2 +1,142 @@
-# paps
-News Room challenge
+# Pap News 
+
+Welcome to Pap News - _Spread the knowledge!_
+
+## Description
+Week 8 Project of Craft Academy cohort January 2018.
+
+PapsNews is a newsagency Web Application used to share your local and international news directly on your device screen.  
+
+Once you land on the Homepage (https://....herokuapp.com/), you need to Signup in order to have access to the web contents
+(click on sign up in the navbar), you will automatically have 7 days free trial, at the end of which you can become a subscriber for a minimum period of 12 months.
+The registration process requires you to enter your name, surname, password, password confirmation and your e-mail address. 
+
+
+## Getting Started
+
+Interested in our codes?
+You can clone our repo!
+
+### Prerequisites
+
+Install Ruby 2.4.1
+ 
+```
+rvm install 2.4.1
+```
+
+Install Ruby on Rails 5.2
+```
+gem install rails -v 5.2
+```
+### Installing
+
+A step by step series of examples that tell you have to get a development env running
+
+Say what the step will be
+
+```
+Give the example
+```
+
+And repeat
+
+```
+until finished
+```
+
+End with an example of getting some data out of the system or using it for a little demo
+
+## Testing
+
+* Cucumber
+* RSpec
+* Pry
+* Factory Bot
+* Launchy
+* Shoulda Matchers
+* Coveralls
+* Travis CI
+
+## Acceptance Tests
+
+Acceptance test are written in Cucumber using Capybara commands. 
+
+Features are stored in ./features
+
+To run the acceptance test suite, execute Cucumber in you terminal:
+
+```
+cucumber
+```
+
+
+
+### Unit Tests 
+
+Unit test are written in rspec
+
+```
+rspec
+```
+
+### And coding style tests
+
+Explain what these tests test and why
+
+```
+Give an example
+```
+
+## Deployment
+
+Add additional notes about how to deploy this on a live system
+
+
+## Technologies
+
+Ruby on Rails
+Haml
+Bootstrap
+Sass
+Active Admin 
+Stripe
+Devise
+Pundit
+i18n
+Active Storage
+
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/your/project/tags). 
+
+## Authors
+
+* **Antonella Morittu**: https://github.com/AntonellaMorittu
+* **David Boll**: https://github.com/davidboll
+* **Aiden Jubilen**: https://github.com/AKidd95
+* **Isabelle Lidberg**: https://github.com/isabellelii
+* **Karl-Magnus Thorsnes**: https://github.com/kmthorsnes
+
+## Contributors
+
+Our restless coaches supported us in this project:
+
+* Thomas Ochman https://github.com/tochman
+* Raoul Diffouo https://github.com/diraulo
+* Faraz Naeem https://github.com/faraznaeem
+* Magnus Þór Ársælsson https://github.com/magnus-thor
+* Olver Ochman ...
```

Comment:
Change Olver to Oliver

Reply:
I removed the incomplete info, it will be added tomorrow.